cmd/main.go

```diff
@@ -201,12 +204,21 @@ func setupManager(flagVar *flags.FlagVar, cacheOptions cache.Options, scheme *ma
 	go cleanupStoredVersions(flagVar.DropCrdStoredVersionMap, mgr, setupLog)
 	go scheduleMetricsCleanup(kymaMetrics, flagVar.MetricsCleanupIntervalInMinutes, mgr, setupLog)
 
+	setupIstioGatewaySecretRotation(config, kcpClient, setupLog)
```

Comment:
I would prefer put the goruntine declaration here so that aligns with above method and has a clear view that all of them are go rountine.

Reply:
what I mean is just like this 
```
	go cleanupStoredVersions(flagVar.DropCrdStoredVersionMap, mgr, setupLog)
	go scheduleMetricsCleanup(kymaMetrics, flagVar.MetricsCleanupIntervalInMinutes, mgr, setupLog)

	go setupIstioGatewaySecretRotation(config, kcpClient, setupLog)
}

func setupIstioGatewaySecretRotation(config *rest.Config, kcpClient *remote.ConfigAndClient, setupLog logr.Logger) {
	kcpClientset := kubernetes.NewForConfigOrDie(config)
	gatewaySecretHandler := gatewaysecret.NewGatewaySecretHandler(kcpClient)
	gatewaysecret.WatchChangesOnRootCertificate(kcpClientset, gatewaySecretHandler, setupLog)
}
```